Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -148,9 +203,44 @@ public void Validate()
         public string UpdateUrl { get; set; } = "";
 
         /// <summary>
-        /// Sets a custom mouse cursor.
+        /// Sets a sprite (must end in .png) for the game client's cursor.
         /// </summary>
-        public string MouseCursor { get; set; } = "";
+        public string ClientCursorSprite { get; set; } = "";
```

Comment:
Don't rename this, it's a breaking configuration change and this is going into a minor version